packages/ocp-build/ocp-build.1.99.21/opam

```diff
@@ -16,7 +16,7 @@ depends: [
   "ocamlfind"
   "cmdliner" {>= "1"}
   "re" {>= "1.7.3"}
-  "seq" { ocaml:version < "4.07.0"}
+  "seq" { ocaml < "4.07.0"}
```

Comment:
What about 
```suggestion
  ("seq" | "ocaml" {>= "4.08"})
```

Reply:
Thanks, but already fixed... normally :-) (thx to @AltGr for the trick)